src/iterative_sorting/iterative_sorting.py

```diff
@@ -5,25 +5,32 @@ def selection_sort( arr ):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
-
+        # (hint, can do in 3 loc)
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
         # TO-DO: swap
-
-
-
-
+        temp = arr[cur_index]
+        arr[cur_index] = arr[smallest_index]
```

Comment:
you can also do something like `arr[cur_index] , arr[smallest_index] =  arr[smallest_index] , arr[cur_index]`